internal/tiltfile/docker.go

```diff
@@ -325,34 +325,40 @@ func (s *tiltfileState) reposToDomain(image *dockerImage) []model.LocalGitRepo {
 		})
 	}
 
+	return result
+}
+
+func (s *tiltfileState) reposToDomain(image *dockerImage) []model.LocalGitRepo {
```

Comment:
this function name/signature doesn't make much sense to begin with, less so now that we may be handling multiple types of manifests. maybe `reposForImage`?